modules/ROOT/pages/api-led-design.adoc

```diff
@@ -0,0 +1,159 @@
+= Build an API Tutorial: Design an API Specification
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To design an API, assess the purpose and requirements:
+
+* Identify the type of API. Is it a simple API, part of an integration, or part of a back-end system?
+* Understand the data flows: one-way, two-way, or more.
+* Explore the security requirements.
+
+Once the scope and flow of your integration project is understood,
+take the first step and define an API specification in RAML (or import to RAML from OAS).
+Then, develop the apps that access the API resources.  
+
+== API Specifications and APIs
+An API is a published interface to a resource that anyone with the correct permissions
+and properly structured request can access. 
+
+An API specification details the functional and expected behavior of an API, 
+as well as the fundamental design philosophy and supported data types.
+It contains both documentation and API definitions 
+to create a contract that humans and software can read. 
+
+MuleSoft provides tools that make it easy to create an API specification that you can then share with your team or publicly.
+The API specification increases adoption and speeds up project completion. 
+
+== Step 1: Explore Existing API Specifications
+If you look at existing API specifications before writing your own, you can learn how other people have approached similar problems. You can also check whether an asset with the same objectives is not already in development, and reuse it if appropriate.
+
+Looking for an API specification that already does what you need is easy:
+
+* Look at the public Exchange. You’ll see some of the most popular API specifications, connectors, and other assets displayed.
++
+link:https://www.mulesoft.com/exchange["Take me there^", role="button-primary"]
+ +
+Select *REST APIs* from the *Type* dropdown to display only REST API specifications. Click on any specification to see the data types and the HTTP requests defined for the API. 
++
+* To look at just the items in your organization (account) Exchange, 
+log in to your Anypoint Platform account (your account).
++
+link:https://anypoint.mulesoft.com/login["Take me there^", role="button-primary"] 
+
+== Step 2: Create Your API Specification
+In link:https://anypoint.mulesoft.com/login/["Anypoint Platform^", role="button-primary"]:
+
+. Click the blue button under Design Center to open the API Designer.
+. Click *Create* to open the API Designer editor.
+. Click *Create API specification*.
+. Enter Hello-World for *Name* and leave the other default values.
```

Comment:
```suggestion
. Type `Hello-World` for *Name* and leave the other default values.
```

Reply:
style guide says to use bold mono. For accessibility, I'd rather avoid specifying the control "type".

Reply:
So you are changing Hello-World here to bold, as well as mono? 
Isn't ` ` mono?
So, this would be \*`Hello-World`*\?

Reply:
so, there are labels and the actual file and they are different. The API spec name is different from the project name...